example_buffer.py

```diff
@@ -68,20 +68,18 @@ def parallel_fill(self, games, threads=8):
         """ games is a list of .tfrecord.zz game records. """
         games.sort(key=os.path.basename)
         # A couple extra in case parsing fails
-        max_games = self.max_size // self.samples_per_game + 10
+        max_games = (self.max_size // self.samples_per_game) + 480
         if len(games) > max_games:
             games = games[-max_games:]
```

Comment:
you can do
`games = games[-max_games:]` without the conditional and it will just work.

Reply:
ack.